import-service/src/lambda/import-file-parser/import-file-parser.ts

```diff
@@ -19,12 +45,19 @@ export const importFileParser: S3Handler = (event) => {
       };
 
       const fileStream = s3.getObject(params).createReadStream();
+      const csvStream = csvParser({
+        mapValues: ({ value }) => {
+          return isNaN(value) ? value : Number(value)
+        }
+      });
+      const sqsStream = new SQSStream(sqs);
 
-      fileStream.pipe(csvParser())
-        .on('data', data => {
-          console.log(`importFileParser has processed '${source}' data chunk`);
-          console.log(data);
+      s3.getObject(params).createReadStream()
```

Comment:
You man use [finished](https://nodejs.org/dist/latest-v15.x/docs/api/stream.html#stream_stream_finished_stream_options_callback) to get rid of multiple `error` event handlers.
In conjunction with [promisify](https://nodejs.org/dist/latest-v15.x/docs/api/util.html#util_util_promisify_original) it allows writing code in "synchronous" style.